### PR TITLE
Spanish Translation + .lat episodes support (South Park)

### PR DESCRIPTION
**Pull request by NOTPIES**
Title says it, also fixed an mistake on the url's.

### CODE REVIEW

File: websites/S/South Park/dist/metadata.json [hunk 1]
```diff
@@ -4,17 +4,24 @@
     "name": "SlowLife.",
     "id": "374905512661221377"
   },
+  "contributors": [
+    {
+      "name": "notpies",
+      "id": "715693234474450955"
+    }
+  ],
```
Comment (Slowlife01) [inline websites/S/South Park/dist/metadata.json:12]: You didn't make 25% changes hence you can't add yourself

File: websites/S/South Park/presence.ts [hunk 2]
```diff
@@ -16,7 +16,14 @@ presence.on("UpdateData", async () => {
...
+	 
+	let sitepath = "episodes";
+	let site = "https://www.southparkstudios.com/";
+	if (path.includes("/episodios/")) {
+		sitepath = "episodios";
+		site = "https://www.southpark.lat/";
+	}
     const EpAndSeason = SouthParkData.children[0].props.title.text
         .split(" - ")[1]
         .match(/([1-9]?[0-9]?[0-9])/g),
```
Comment (Slowlife01) [inline websites/S/South Park/presence.ts:29]:
```suggestion
    const Url = path.includes("/episodios/") ? "https://www.southpark.lat/episodios" : "https://www.southparkstudios.com/episodes",
      EpAndSeason = SouthParkData.children[0].props.title.text
        .split(" - ")[1]
        .match(/([1-9][0-9]?[0-9]?)/g),
```

File: websites/S/South Park/presence.ts [hunk 3]
```diff
@@ -46,9 +53,10 @@ presence.on("UpdateData", async () => {
       presenceData.buttons = [
         {
           label: "Watch Episode",
-          url: `https://www.southparkstudios.com/episodes/${
+		  url: `${site + sitepath}/${
```
Comment (Slowlife01) [inline websites/S/South Park/presence.ts:56]:
```suggestion
		      url: `${Url}/${
```

**Comment by Slowlife01** [comment]
Show proof of it working afterwards

**Comment by NOTPIES** [comment]
image

using .lat site

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**